alma8/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM @BASE_IMAGE_NAME@
-RUN dnf install -y @DEFAULT_PACKAGES@ automake bash bzip2 bzip2-libs bzip2-devel \
+RUN dnf install -y bash && bash @SCRIPT@ BASE &&\
```

Comment:
@aandvalenzuela , why using `@SCRIPT@`? just add
```
ADD rpm-pkg-info.sh /tmp/rpm-pkg-info.sh
```
after `FROM ` command. and then use `/tmp/rpm-pkg-info.sh` to run it.

You can move the  `rpm-pkg-info.sh` one level up but still need to create a symlink in the repo directory e.g.
```
cms-docker/etc/rpm-pkg-info.sh
cms-docker/alma8/rpm-pkg-info.sh -> ../etc/rpm-pkg-info.sh  (symlink)
```